include/common_structs.h

```diff
@@ -1436,22 +1443,17 @@ typedef struct ShopSellPriceData {
 } ShopSellPriceData; // size = 0xC
 
 typedef struct GameStatus {
-    /* 0x000 */ u32 currentButtons;
-    /* 0x004 */ u32 altCurrentButtons; /* input used for batte when flag 80000 set */
+    /* 0x000 */ u32 currentButtons[2];
```

Comment:
What motivated this change?

Reply:
https://github.com/pmret/papermario/pull/733/files#diff-a4f821cdff36382a835cadb6c536f46a4e885dd4f3bb3081444c7a7723259ac5R572

see the beginning of _use_partner_ability